crates/api_models/src/payments.rs

```diff
@@ -4892,6 +4893,14 @@ pub struct PaymentsConfirmIntentResponse {
 
     /// Error details for the payment if any
     pub error: Option<ErrorDetails>,
+
+    /// The transaction authentication can be set to undergo payer authentication. By default, the authentication will be marked as NO_THREE_DS
+    #[schema(value_type = Option<AuthenticationType>, example = "no_three_ds")]
+    pub authentication_type: Option<api_enums::AuthenticationType>,
+
+    /// The authentication type applied for the payment
+    #[schema(value_type = AuthenticationType, example = "no_three_ds", default = "three_ds")]
+    pub applied_authentication_type: api_enums::AuthenticationType,
```

Comment:
Nit: Is the `default` actually `three_ds` here? Or should we consider removing it?